<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_motor</class>
 <widget class="QDialog" name="Add_motor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fff</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>246</width>
     <height>213</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_Fabricante">
        <property name="text">
         <string>Fabricante</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Edit_Fabricante"/>
      </item>
      <item>
       <widget class="QLabel" name="label_CurrMax">
        <property name="text">
         <string>Corrente Máx(A)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Edit_CurrMax"/>
      </item>
      <item>
       <widget class="QLabel" name="label_rot">
        <property name="text">
         <string>Rotação Máx(rpm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Edit_Rot"/>
      </item>
      <item>
       <widget class="QLabel" name="label_preco">
        <property name="text">
         <string>Preço(R$)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Edit_Preco"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_Tensao">
        <property name="text">
         <string>Tensão(V)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Edit_Tensao"/>
      </item>
      <item>
       <widget class="QLabel" name="label_CurrMin">
        <property name="text">
         <string>Corrente Mín(A)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Edit_CurrMin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_Torq">
        <property name="text">
         <string>Torque Máx (kgf.mm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Edit_Torque"/>
      </item>
      <item>
       <widget class="QPushButton" name="BtnOK">
        <property name="styleSheet">
         <string notr="true">background-color:#c73232;
color:#fff;</string>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_titulo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fff;
color:#c73232;</string>
   </property>
   <property name="text">
    <string>Novo motor:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
